packages/storybook/stories/toggletip.mdx

```diff
@@ -0,0 +1,55 @@
+import { Toggletip } from '@equisoft/design-elements-react';
+import { ArgTypes, Meta, Canvas, Stories, Story } from '@storybook/blocks';
+import * as ToggletipStories  from './toggletip.stories';
+
+<Meta title="Components/Toggletip/Doc" />
+
+# Toggletip
+
+## Table of contents
+    1. [Definition](#definition)
+    2. [Usage](#usage)
+    3. [Variants](#variants)
+    4. [Properties](#properties)
+
+## Definition
+A Toggletip is a disclosure component used to display additional information about a target element, without the user losing the context of the page [2].
+<Canvas of={ToggletipStories.Default} />
+
+## Usage
+
+### When to use
+- Use sparingly and only when the information cannot be presented to the user in any other way (e.g. directly on the page)[7].
+- Provides additional information about a page element (feature, action, content, etc) [1].
+- Use to display instructions or tips that help the user interact with the user interface more effectively.
+- Use to provide brief, useful content.
+- Use for secondary information [4]. Use it when it is not essential for the user to act on the information or complete a task [2].
+- Use when you don‘t want to let the user lose context, but still want to provide additional useful unique information about referring elements. [2].
+- Highlight a new feature or important feature that extends or improves the user's experience on the page [8].
+
+### When not to use
+- Avoid using it as a workaround for poor user interface design, including confusing organization or presentation of information, inconsistent labeling, difficult-to-use or understand functions, or any other issue that makes the user interface hard to use or not user-friendly.
```

Comment:
"_that makes the user interface hard to use or not user-friendly._" si l'interface est hard to use, est-ce que nous pouvons dire qu'elle est "_not user friendly_"? Ma question est plus, est-ce que les deux veulent dire la même chose?